packages/ref-doc/src/utils/markdown.ts

```diff
@@ -23,7 +23,7 @@ export function inlinecode(code: string) {
 }
 
 function escapeMarkdownTable(text: string) {
-  return text.replace(/(\|)/g, "\\$1");
+  return text.replace(/(\|)/g, "\\$1").replace("\n", "<br />");
```

Comment:
This will only replace the first `\n`, won't it? Oops. Let me see if I can add a test.